test/integration/targets/setup_pexpect/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+- name: Install pexpect
+  pip:
+    name: pexpect
```

Comment:
This is a simple enough operation it can be done inline in the test instead of as a dependency.

Reply:
ok thank you